### PR TITLE
Fix build.sh failing

### PR DESCRIPTION
**Pull request by pmnoxx**
Bare wasm contract is 35k on my system, which causes script to fail, due to `dd` value being negative.

```

++ filesize target/wasm32-unknown-unknown/release/test_contract.wasm
++ local file=target/wasm32-unknown-unknown/release/test_contract.wasm
+++ stat -c%s target/wasm32-unknown-unknown/release/test_contract.wasm
++ size=39792
++ '[' 0 -eq 0 ']'
++ echo 39792
++ return 0
+ bare_wasm=39792
+ echo 39792
39792
++ expr 10240 - 39792
+ dd if=/dev/urandom of=./res/payload bs=-29552 count=1
dd: invalid number: ‘-29552’
```

I also switched `#!/bin/bash` to `#!/usr/bin/env bash`, which should be portable and work for both linux and macOS.

### CODE REVIEW

File: runtime/runtime-params-estimator/test-contract/build.sh [hunk 1]
```diff
@@ -29,6 +29,12 @@ echo ${bare_wasm}
 # bloat its size to the given values.
 
 # 10KiB
+
+# hotfix for people building cargo with debug flags, causing binary to be 35kb
```
Comment (willemneal) [inline runtime/runtime-params-estimator/test-contract/build.sh:33]: Shouldn't users always build release when testing performance?

Comment (pmnoxx) [reply runtime/runtime-params-estimator/test-contract/build.sh:33]: I'm running release build with debug symbols on, so the performance shouldn't be affected. It's useful for debugging though.

We need files produced by this build script for running `cargo test`. 
That wasn't the case in the past. 
By default all tests run when you execute `cargo test`, you have no control over it. So this is a regression.

Comment (pmnoxx) [reply runtime/runtime-params-estimator/test-contract/build.sh:33]: I would also be fine if we don't merge `build.sh` fix, but instead disable broken tests mentioned in https://github.com/near/near-ops/issues/440
Alternatively, we can commit 
```

49 | |     CODE_10K => "test-contract/res/stable_small_contract.wasm",
50 | |     CODE_100K => "test-contract/res/stable_medium_contract.wasm",
51 | |     CODE_1M => "test-contract/res/stable_large_contract.wasm",
``` to the repo

Comment (matklad) [reply runtime/runtime-params-estimator/test-contract/build.sh:33]: >I'm running release build with debug symbols on, so the performance shouldn't be affected. It's useful for debugging though.

I think there's a bit of confusion here. There are two "debug" related cargo toml fields. `debug` controls debug symbols (`-g`). `debug-assertions` controls `debug_assert`. The `lazy_static_include` crate uses the latter to switch between compile time and runtime inclusion.

Comment (matklad) [reply runtime/runtime-params-estimator/test-contract/build.sh:33]: https://github.com/near/nearcore/pull/4125 fixes the compilation error.

Comment (pmnoxx) [reply runtime/runtime-params-estimator/test-contract/build.sh:33]: I'll close this PR then.

**Pull request closed without merging**